reporter.go

```diff
@@ -179,8 +180,9 @@ type reporterQueueItem struct {
 type remoteReporter struct {
 	// These fields must be first in the struct because `sync/atomic` expects 64-bit alignment.
 	// Cf. https://github.com/uber/jaeger-client-go/issues/155, https://goo.gl/zW7dgq
-	queueLength int64
-	closed      int64 // 0 - not closed, 1 - closed
+	queueLength  int64
+	closed       int64 // 0 - not closed, 1 - closed
+	droppedCount int64
```

Comment:
It looks like `droppedCount` isn't written to. Was the intention to increment this in `func (r *remoteReporter) Report(span *Span)`, or do you intend to implement this in another diff or perhaps it's somehow done elsewhere which I've missed?

Reply:
It is incremented in SpansDroppedFromQueue